09_dynamic_programming/python/longest_common_subsequence.py

```diff
@@ -0,0 +1,16 @@
+wordA='blue'
+wordB='clues'
+lstA = list(wordA)
+lstB = list(wordB)
+#create a 4*5 matrix 
+cell = [([0]*len(lstB)) for i in range(len(lstA))]
+
+for x in range(1,len(lstA)):
+	for y in range(1,len(lstB)):

```

Comment:
Could you change indentation to 2 spaces to keep it consistent with the rest of the python examples?